app-modules/portal/src/Filament/Pages/ManagePortalSettings.php

```diff
@@ -219,4 +228,30 @@ public function form(Form $form): Form
                     ->visible(fn (Get $get) => $get('knowledge_management_portal_enabled')),
             ]);
     }
+
+    public function save(): void
+    {
+        $this->callHook('beforeValidate');
+
+        $data = $this->form->getState();
+
+        $this->callHook('afterValidate');
+
+        $data = $this->mutateFormDataBeforeSave($data);
+
+        $this->callHook('beforeSave');
+
+        $settings = app(static::getSettings());
+
+        if (FeatureFlag::GDPRNewBannerText->active()) {
+            $settings->cookie_gdpr_banner_text = $data['cookie_gdpr_banner_text'];
+        }
+
+        $settings->fill($data);
+        $settings->save();
+
+        $this->callHook('afterSave');
+
+        $this->getSavedNotification()?->send();
+    }
```

Comment:
Is this needed? The `cookie_gdpr_banner_text` field would not be present if the Feature Flag was not active so it wouldn't be present in the data.